foodsaving/cases/templates/new_conflict_resolution_case.mjml

```diff
@@ -0,0 +1,38 @@
+<mjml>
+    <mj-head>
+        <mj-include path="../../base/templates/_base-head.mjml"/>
+        <mj-title>{% trans creator_name=created_by.display_name, affected_user_name=affected_user.display_name %}{{ creator_name }} opened a discussion about a conflict with {{ affected_user_name }}{% endtrans %}</mj-title>
```

Comment:
Hmm... I don't love it. But I also don't know if any of these are better:
- 'X started a conflict resolution process about Y'
- 'A conflict resolution process was started about Y by X'